src/SLIP10Node.ts

```diff
@@ -503,15 +530,20 @@ export class SLIP10Node implements SLIP10NodeInterface {
       'SLIP10Node can only be constructed using `SLIP10Node.fromJSON`, `SLIP10Node.fromExtendedKey`, `SLIP10Node.fromDerivationPath`, or `SLIP10Node.fromSeed`.',
     );
 
+    assert(
+      privateKey !== undefined || publicKey !== undefined,
```

Comment:
How can this happen now?

Reply:
I don't think it can, but this is just for sanity.